src/core/CCIO.ml

```diff
@@ -222,11 +222,7 @@ let with_in_out ?(mode=0o644) ?(flags=[Open_creat]) filename f =
 let tee funs g () = match g() with
```

Comment:
As a side-note, should this function be in CCIO?

Reply:
it's useful to apply several functions to an input stream, but agreed, it's a bit specialized.
it's there so I think we should keep it, though.